### PR TITLE
[New] Add tablesaw@3.0.9  w/ git auto-update

### LINKED ISSUE #8447: [Request] Add Tablesaw
**Library name:** Tablesaw
**Git repository url:** https://github.com/filamentgroup/tablesaw/
**npm package url(optional):** https://www.npmjs.com/package/tablesaw
**License(s):** Copyright (c) 2013 Filament Group
**Official homepage:** https://www.filamentgroup.com/
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
Use web-minify-helper for files minifying
close #8447, cc @zachleat

# Pull request template for new library

Pull request for issue: #8447 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/filamentgroup/tablesaw
* Official website (optional, not the repository): https://filamentgroup.github.io/tablesaw/demo/kitchensink.html"
* NPM package url (optional): https://www.npmjs.com/package/tablesaw
* License and its reference: MIT
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 198
  * Count of stars: 5104
  * Count of forks: 419
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week: 1489
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [x] Close corresponding issue in commit message
* [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/tablesaw/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "url": "git://github.com/filamentgroup/tablesaw.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/filamentgroup/tablesaw.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(*.scss|*.txt|*.html)",
```
Comment (extend1994) [inline ajax/libs/tablesaw/package.json]: Let's use `!(dependencies){,**/!(*.scss|*.txt|*.html)}` to minimize number of the rules.

**Comment by extend1994** [comment]
Update to exclude icons in the old versions from https://github.com/filamentgroup/grunticon/tree/master/example/output

**Comment by extend1994** [comment]
@PeterDaveHello I found that the pattern will fail to exclude `tablesaw.stackonly.scss` in v0.0.1  ~ v0.1.6 after I merged this PR... I suppose they won't be added automatically due to different file structure? Sorry for the mistake!

**Comment by PeterDaveHello** [comment]
@extend1994 I'll taka care of it, no worries.

**Comment by extend1994** [comment]
@PeterDaveHello Thanks a lot!

**Pull request merged**